src/core/namespace.c

```diff
@@ -3237,7 +3237,7 @@ static const char* const private_tmp_table[_PRIVATE_TMP_MAX] = {
 DEFINE_STRING_TABLE_LOOKUP_WITH_BOOLEAN(private_tmp, PrivateTmp, PRIVATE_TMP_CONNECTED);
 
 static const char* const private_users_table[_PRIVATE_USERS_MAX] = {
-        [PRIVATE_USERS_OFF]      = "off",
+        [PRIVATE_USERS_OFF]      = "no",
```

Comment:
hmm, what's the rationale of this change? 

I am not a fan of enums whose string representation is different from their name. i.e. here now "off" vs. "no". Either rename neither or both. But I don#t grok the rationale here?